tools/pywebsocket/mod_pywebsocket/standalone.py

```diff
@@ -348,8 +348,8 @@ def recv(self, bufsize, flags=0):
 
         try:
             return self._connection.recv(bufsize)
-        except OpenSSL.SSL.SysCallError, (err, message):
```

Comment:
I have been unable to figure out what this syntax meant; would appreciate a second look from someone who knows.

Reply:
Python 2 exceptions are like tuples:

```
Exception(0,1,2,3)[2]
// 2
```

Python 3 does not allow this and requires using `.args`, which gives a tuple.